test/tools/libtesteth/TestOutputHelper.cpp

```diff
@@ -124,14 +124,49 @@ bool pathHasTests(boost::filesystem::path const& _path)
     return false;
 }
 
-void TestOutputHelper::checkUnfinishedTestFolders()
+string TestOutputHelper::detectFilterForMinusTArgument()
 {
     // -t SuiteName/caseName   parse caseName as filter
     // rCurrentTestSuite is empty if run without -t argument
+    Options const& opt = Options::get();
+    if (opt.rCurrentTestSuite.empty())
+        return string();
+
     string filter;
-    size_t pos = Options::get().rCurrentTestSuite.find('/');
-    if (pos != string::npos)
-        filter = Options::get().rCurrentTestSuite.substr(pos + 1);
+    // -t SuiteName/subSuiteName/caseName
+    // if (SuiteName/subSuiteName == -t argument) then filter is empty
+    bool thereisPathEqualToSuite = false;
+    for (auto const& allTestsIt : m_finishedTestFoldersMap)
+    {
+        // Remove Filler from folder name so it equal to -t argument
+        string parent = allTestsIt.first.parent_path().stem().string();
+        size_t index = parent.find("Filler");
+        if (index == std::string::npos)
+            break;
+        parent = parent.replace(index, 6, "");
```

Comment:
```suggestion
        parent = parent.substr(0, index);
```

Reply:
hmm. I think it wont work. `BlockchainTestsFiller/sometestsFoder1`
I need to get  `BlockchainTestsFiller/sometestsFoder1`
but will get `BlockchainTests`

Reply:
hm. another hardcode on folder names.